install_config/aggregate_logging.adoc

```diff
@@ -473,6 +473,62 @@ $ oc delete secret logging-fluentd logging-elasticsearch \
     logging-kibana-ops-proxy
 ----
 
+
+== Upgrading the EFK Stack From 3.1 to 3.2
```

Comment:
How about I add sub-sections and note that upgrading to Origin 1.2 and OSE 3.2. We'll add a new section when the next major upgrade procedure arrives.